### PR TITLE
Fix duplicate styled-components warning

### PR DESCRIPTION
**Pull request by srallen**
Package: lib-react-components, lib-classifier

Describe your changes:
Updates webpack configs to resolve duplicate styled-components warnings. Removes workbox configs from classifier, which I had forgotten to do in my PR that disabled the service worker.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

**Comment by srallen** [comment]
#532 is branched off this, so it'd be nice to review that one first and merge into this. Then merge all this into master so there isn't a merge conflict.

File: packages/lib-classifier/webpack.dev.js [hunk 1]
```diff
@@ -52,10 +47,13 @@ module.exports = {
...
   plugins: [
     EnvironmentWebpackPlugin,
-    HtmlWebpackPluginConfig,
-    WorkboxPluginConfig
+    HtmlWebpackPluginConfig
   ],
   resolve: {
+    alias: {
+      // adjust this path as needed depending on where your webpack config is
+      'styled-components': path.resolve('./node_modules/styled-components')
```
Comment (rogerhutchings) [inline packages/lib-classifier/webpack.dev.js]: Won't this resolve to the version of styled-components in this library, not the consuming app?

File: packages/lib-classifier/webpack.dist.js [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-  swSrc: './src/workers/queue.js'
-})
-
+// The peer deps external plugin doesn't work with scoped packages
+// so we add them explicitly to webpack's externals config
+// see: https://github.com/Updater/peer-deps-externals-webpack-plugin/issues/5
 module.exports = {
   entry: './src/components/Classifier/index.js',
+  externals: [
+    /^(@zooniverse\/grommet-theme\/.*)$/i,
```
Comment (rogerhutchings) [inline packages/lib-classifier/webpack.dist.js]: We could have one regex for everything under the `@zooniverse` namespace

File: packages/lib-classifier/webpack.dev.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/lib-classifier/webpack.dev.js]: This is the dev version, so I think it's fine.

For production, though, should we bite the bullet and hoist shared modules like `styled-components` and `react` to the top-level `node_modules` directory?

Comment (srallen) [reply packages/lib-classifier/webpack.dev.js]: Yeah it's for the dev version locally to this lib. We definitely need to move to hoisting at some point.

File: packages/lib-classifier/webpack.dist.js (hunk 2 above)
Comment (srallen) [reply packages/lib-classifier/webpack.dist.js]: We could, though the issue workaround showed specific regexes to match namespaced packages, so I just followed that. I'm not sure if there's a reason for that.

**Comment by srallen** [comment]
Merge conflicts anyway :sadpanda:. I'll fix these when I'm done with #516

**Comment by srallen** [comment]
@rogerhutchings the merge conflicts are resolved and this can have a final review.

File: packages/app-project/pages/_app.js [hunk 3]
```diff
@@ -69,11 +70,12 @@ export default class MyApp extends App {
 
   render () {
     const { Component, pageProps, theme } = this.props
+    const mergedThemes = merge({}, base, theme)
```
Comment (rogerhutchings) [inline packages/app-project/pages/_app.js]: We're careful on this one, so I don't worry about it unduly, but we should probably freeze this at some point

**Comment by rogerhutchings** [comment]
It looks like there are styling regressions - I'm going to branch off and try and fix them.

**Comment by rogerhutchings** [comment]
Aha - there aren't, but the footer _is_ rendering outside of the Grommet context. I think this should be fixed by `copy-libs`

**Pull request merged**